plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -590,26 +591,31 @@ open class DefaultPageCreator(
                             availablePlatforms.forEach { platform ->
                                 val possibleFallbacks = sourceSets.getPossibleFallbackSourcesets(platform)
                                 seeAlsoTags.forEach { (_, see) ->
-                                    (see[platform] ?: see.fallback(possibleFallbacks))?.let {
+                                    (see[platform] ?: see.fallback(possibleFallbacks))?.let { seeTag ->
                                         row(
                                             sourceSets = setOf(platform),
                                             kind = ContentKind.Comment,
                                             styles = this@group.mainStyles,
                                         ) {
-                                            it.address?.let { dri ->
+                                            seeTag.address?.let { dri ->
+                                                // only declaration links are expected to be fully qualified
+                                                val linkText = dri.takeIf { it.target is PointingToDeclaration }
+                                                    ?.let { seeTag.name.removePrefix("${it.packageName}.") }
+                                                    ?: seeTag.name
```

Comment:
Turns out, `@see` can also be used with local params (we have tests for it) and generics. And with functions, of course, so using `classNames` was wrong here, my bad